Document/0x06b-Basic-Security-Testing.md

```diff
@@ -1332,150 +1339,6 @@ The last step would be to set the proxy globally on your iOS device:
 
 Open Safari and go to any webpage, you should see now the traffic in Burp. Thanks @hweisheimer for the [initial idea](https://twitter.com/hweisheimer/status/1095383526885724161 "Port Forwarding via USB on iOS")!
 
-##### Using Frida
```

Comment:
is it correct that these materials will be moved?

Reply:
yes, this is all about tracing and hooking which is all now in 6c in the Dynamic Analysis section(same as in 5c now)